### ISSUE TITLE
Build failure with GHC 9.0

### ISSUE THREAD

**Issue by Bodigrim** [fizruk/telegram-bot-simple#101]
```
$ cabal build -w ghc-9.0
src/Telegram/Bot/API/Types.hs:1027:30: error:
    • No instance for (ToJSON PassportElementError)
        arising from a use of ‘encodeToLazyText’
    • In the second argument of ‘(.)’, namely ‘encodeToLazyText’
      In the expression: TL.toStrict . encodeToLazyText
      In an equation for ‘toUrlPiece’:
          toUrlPiece = TL.toStrict . encodeToLazyText
     |
1027 |   toUrlPiece = TL.toStrict . encodeToLazyText
     |                              ^^^^^^^^^^^^^^^^

src/Telegram/Bot/API/Types.hs:1030:30: error:
    • No instance for (ToJSON PassportElementError)
        arising from a use of ‘encodeToLazyText’
    • In the second argument of ‘(.)’, namely ‘encodeToLazyText’
      In the expression: TL.toStrict . encodeToLazyText
      In an equation for ‘toUrlPiece’:
          toUrlPiece = TL.toStrict . encodeToLazyText
     |
1030 |   toUrlPiece = TL.toStrict . encodeToLazyText
     |                              ^^^^^^^^^^^^^^^^

src/Telegram/Bot/API/Types.hs:1146:50: error:
    • No instance for (ToJSON MessageEntity)
        arising from a use of ‘gtoJSON’
    • In the expression: gtoJSON
      In an equation for ‘toJSON’: toJSON = gtoJSON
      In the instance declaration for ‘ToJSON InputMediaGeneric’
     |
1146 | instance ToJSON InputMediaGeneric where toJSON = gtoJSON
     |                                                  ^^^^^^^

src/Telegram/Bot/API/Types.hs:1156:55: error:
    • No instance for (ToJSON MessageEntity)
        arising from a use of ‘encodeToLazyText’
    • In the second argument of ‘($)’, namely ‘encodeToLazyText t’
      In the second argument of ‘Input’, namely
        ‘(TL.toStrict $ encodeToLazyText t)’
      In the expression:
        Input "caption_entities" (TL.toStrict $ encodeToLazyText t)
     |
1156 |         \t -> Input "caption_entities" (TL.toStrict $ encodeToLazyText t)
     |                                                       ^^^^^^^^^^^^^^^^^^
```

Cf. #46.

**Comment by swamp-agr** [comment]
Fix released in https://hackage.haskell.org/package/telegram-bot-simple-0.4.2.
Thank you for reporting the issue.

**Comment by Bodigrim** [comment]
@swamp-agr this strikes back in `telegram-bot-simple-0.5`. May I suggest setting up a CI via `haskell-ci` to prevent such issues in future again?

**Comment by Bodigrim** [comment]
As a Hackage trustee I made a revision to prohibit unbuildable configuration:
https://hackage.haskell.org/package/telegram-bot-simple-0.5/revisions/

**Comment by swamp-agr** [comment]
For some reasons I haven't received any notification. 
I will set up Github actions soon.

**Comment by swamp-agr** [comment]
GitHub actions set via #119 and compilation errors fixed via #120.
